sdk/keyvault/keyvault-secrets/test/public/CRUD.spec.ts

```diff
@@ -93,10 +93,14 @@ describe("Secret client - create, read, update and delete operations", () => {
     const secretName = testClient.formatName(
       `${secretPrefix}-${this!.test!.title}-${secretSuffix}`
     );
-    const secretValue = "";
-    const result = await client.setSecret(secretName, secretValue);
+    const emptySecretValue = "";
+    const result = await client.setSecret(secretName, emptySecretValue);
```

Comment:
Why is this necessary? (this is where I intended to send the "but why" gif)

Reply:
`secretValue` here shadows another one in the scope.